lib/worker/base.js

```diff
@@ -176,7 +176,14 @@ const run = async options => {
 
 	try {
 		for await (const ref of (options.require || [])) {
-			await load(ref);
+			if (typeof ref === 'string') {
+				await load(ref);
+			} else if (Array.isArray(ref)) {
+				const [path, options] = ref;
+
+				const mod = await load(path);
+				mod.apply(null, ...options);
```

Comment:
Need to check how this works with a mix of CJs and ESM. You may need to check if `mod` is callable, and if not, access a callable `default`.

Reply:
Thanks @novemberborn  - I'll correct that :)